### DISCUSSION TITLE
Renault Scenic 3 (2015)

### DISCUSSION THREAD

**Discussion by MrTwelveTrees** [cedricp/ddt4all#1195] [General]
Hi, i can not find Scenic 3 on list I am using database https://drive.google.com/file/d/14AQTXRv4Y0mVnWNeZ6seznKPRd-7ZaKm/view On every turial on youtube Scenic 3 is like X95 on the list ?

**Comment by MrTwelveTrees** [comment]
I am traying to deactive TPMS
 I have Rlink (if that is important?)
---> https://ibb.co/kBp5K7S
![Posnetek zaslona 2024-08-13 153232](https://github.com/user-attachments/assets/e46dba8e-73f1-45cd-836a-7c2f8bd6962f)

**Comment by AndyIreland** [comment]
yep you have to do it through BCM - and possibly disable it in your Meter Cluster settings too .

**Comment by MrTwelveTrees** [comment]
Where can i find this ?
BCM/UCH...... not here

ps: TPMS in not displayed on Rlink1 display.

**Comment by AndyIreland** [comment]
set this to without in BCM/UCH:
![image](https://github.com/user-attachments/assets/6b5e9ab7-3bdc-44b1-9931-81404b95d909)


Go to Cluster and set to SANS (Presuming your cluster is B95_L38 - you might have different one)
![image](https://github.com/user-attachments/assets/38b48c0f-0d1e-40d9-8289-b09fe111c5d9)

Then if you have R-Link1 , make sure you can read/write rlink through CAN socket with a modded/rewired 
 Y-cable set to without (presuming your MFD is v5.2 for example) : 
 
 
![image](https://github.com/user-attachments/assets/7933d852-398e-4c30-b34c-fe2b0200a12d)


good luck

**Comment by MrTwelveTrees** [comment]
Thank you !

i had to do only: BCM/UCH and SANS
nothinh on Rlink settings
